homeassistant/components/axis/device.py

```diff
@@ -241,10 +239,12 @@ def disconnect_from_stream(self):
     @callback
     def shutdown(self, event):
         """Stop the event stream."""
+        self.hass.loop.create_task(self.api.vapix.close())
```

Comment:
Why not await it ?

Reply:
Its a callback?

Reply:
Make `shutdown` a coroutine function instead. Then we can await inside it.

Reply:
I didn't know of the new machinations in the background regarding hass_job, nice!

Also: fixed!